numberGame.js

```diff
@@ -0,0 +1,37 @@
+import compareFunction from './compareFunction.js';
+
+const userInput = document.getElementById('enter-guess');
+const submitButton = document.getElementById('button');
+const triesRemaining = document.getElementById('tries-remaining');
+const box = document.getElementById('box');
+const winDisplay = document.getElementById('winDisplay');
+const lostDisplay = document.getElementById('lostDisplay');
+
+let tries = 4;
+
+const correctNumber = Math.round(Math.random() * 20) ;
+
+
+submitButton.addEventListener('click', () => {
```

Comment:
If you guess the right answer on the last guess you still lose the game. You'll want to add a special case for someone guess the right answer on their last try.